src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java

```diff
@@ -26,23 +36,6 @@
 import org.kohsuke.stapler.DataBoundConstructor;
 import org.kohsuke.stapler.DataBoundSetter;
 
-import com.google.common.base.Strings;
```

Comment:
Set your editor to avoid reorganizing imports. Just creates diff churn and merge conflicts. (Ideally we would normalize all imports and enforce this at build time: strict lexicographic sort, no wildcards, no blank lines.)